packages/trader/src/Modules/MT5/Containers/mt5-financial-stp-real-account-signup.jsx

```diff
@@ -209,10 +211,12 @@ class MT5FinancialStpRealAccountSignup extends React.Component {
     };
 
     componentWillUnmount() {
-        sessionStorage.setItem(
-            MT5_PERSONAL_DETAILS_CACHE_KEY,
-            JSON.stringify(this.state.items[index_lookup.MT5PersonalDetailsForm].form_value)
-        );
+        if (this.should_retain_cache) {
```

Comment:
let's fix these when we have time. the flow is not very clear in MT5 account opening from the code as it has been  refactored to the point of losing readability.

Reply:
Sure, will hold on to this for now

Reply:
@reza-fs no need to wait, we can go ahead with this for now